.github/workflows/Build.yml

```diff
@@ -38,93 +38,31 @@ jobs:
         name: shasta-Ubuntu-20.04.tar
         path: shasta-build/shasta-Ubuntu-20.04.tar
 
-
-  ubuntu-18-04:
-    runs-on: ubuntu-18.04
+  ubuntu-20-04-minimal:
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@master
     - name: Install-prerequisites
-      run: sudo ./scripts/InstallPrerequisites-Ubuntu.sh
-    - name: Build-ubuntu-18.04
+      run: sudo ./scripts/InstallPrerequisites-Ubuntu.sh --minimal
+    - name: Build-ubuntu-20.04
```

Comment:
This has the same name as the one for the regular Ubuntu build. I am not sure this is a problem, but isn't it better to have a name that include "minimal", to avoid confusion?

Reply:
It's not a problem because it is under a different top-level Github action. I'll update it to avoid confusion.